include/CFinalBoneHierarchy.h

```diff
@@ -9,6 +9,7 @@
 #include "IGPUMappedBuffer.h"
 #include "quaternion.h"
 #include "irrString.h"
+#include "CBawFile.h"
```

Comment:
If we're including it here, then move CBawFile.h from "source/Irrlicht" to "include"